### PR TITLE
Add optional HTTP pprof server

### PR DESCRIPTION
**Pull request by rdmitr**

### CODE REVIEW

File: models/config/config.go [hunk 1]
```diff
@@ -32,6 +38,15 @@ type AppConfig struct {
 	FrontendUrl        string `envconfig:"FRONTEND_URL"`
 	LogEvents          bool   `envconfig:"LOG_EVENTS" default:"false"`
 	ConnectAlbyAccount bool   `envconfig:"CONNECT_ALBY_ACCOUNT" default:"true"`
+	GoProfilerAddr     string `envconfig:"GO_PROFILER_ADDR"`
+
+	DdProfilerEnabled   bool             `envconfig:"DD_PROFILER_ENABLED" default:"false"`
```
Comment (rolznz) [inline models/config/config.go:43]: Normally datadog can read the environment variables itself and does not require declaring them, is it different here?

**Comment by rolznz** [comment]
@rdmitr any chance could you add some info to the README on how to enable profiling and how to view the profiling data/results?

File: models/config/config.go (hunk 1 above)
Comment (rdmitr) [reply models/config/config.go]: Guess I misunderstood it — kudos to Michael for fixing those!

**Comment by rdmitr** [comment]
Canceling: replaced with https://github.com/getAlby/nostr-wallet-connect-next/pull/366

**Pull request closed without merging**